units/XNA0001/XNA0001_script.lua

```diff
@@ -1,36 +1,41 @@
 local NomadsEffectTemplate = import('/lua/nomadseffecttemplate.lua')
 local Unit = import('/lua/sim/Unit.lua').Unit
 
+---@class XNA0001 : Unit
 XNA0001 = Class(Unit) {
 
     CoverLaunchFx = NomadsEffectTemplate.ACUMeteorCoverLaunch,
     CoverOpenFx = NomadsEffectTemplate.ACUMeteorCoverOpen,
 
+    ---@param self XNA0001
     OnCreate = function(self)
+        local bp = self.Blueprint
+
         Unit.OnCreate(self)
```

Comment:
`self.Blueprint` is initialized in `Unit:OnCreate()`, so the upvalue must come second.
```suggestion
        Unit.OnCreate(self)

        local bp = self.Blueprint
```